hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/lib/db/DBOutputFormat.java

```diff
@@ -122,7 +122,7 @@ public void write(K key, V value) throws IOException {
         key.write(statement);
         statement.addBatch();
       } catch (SQLException e) {
-        e.printStackTrace();
+        throw new RuntimeException("Failed to execute SQL statement for key " + key, e);
```

Comment:
In other places. It is converting the SQLException to IOE.
``
        catch (SQLException ex) {
          throw new IOException(ex.getMessage());
``
Can we keep the same behaviour, will that also solve the problem.

Reply:
Good point. I think it makes sense to keep things uniform so I changed it to `IOException` in https://github.com/apache/hadoop/pull/3671/commits/1e67ceca0cc8459a7b8b98fbb9c65c7294529269